foundation/src/main/java/org/cbio/portal/pipelines/foundation/ClinicalDataWriter.java

```diff
@@ -82,21 +85,28 @@ public String getHeader(){
         for (String key : map.keySet() ) {
             header.add(key);
         }
+        
```

Comment:
right, if the organism was `EBV` the clinical attribute would be called `EBV_STATUS`, and `HPV_STATUS` for `HPV`, etc.